package.json

```diff
@@ -19,6 +19,7 @@
     "node-fetch": "^2.6.0",
     "print-diff": "^0.1.1",
     "styleless-innertext": "^1.1.2",
+    "ts-node": "^8.10.2",
```

Comment:
probably don't need ts-node if manualAnalytics is gone, right?

Reply:
Agree.